src/main/java/groovy/ui/GroovyMain.java

```diff
@@ -223,8 +231,14 @@ static void processArgs(String[] args, final PrintStream out, final PrintStream
         @Option(names = {"-v", "--version"}, versionHelp = true, description = "Print version information and exit")
         private boolean versionRequested;
 
+        @Option(names = {"-cs", "--compile-static"}, description = "Use CompileStatic")
```

Comment:
It's better to have a single-character short option if possible.
That allows picocli to treat it as a POSIX short option, with benefits for parsing and the usage help message layout.

Reply:
Ok, I think it is better to leave only full option names.